src/igvfd/schemas/in_vitro_system.json

```diff
@@ -0,0 +1,195 @@
+{
+    "title": "In Vitro System",
+    "$id": "/profiles/in_vitro_system.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "Schema for submitting a sample grown in a in vitro system.",
+    "type": "object",
+    "required": [
+        "award",
+        "lab",
+        "source",
+        "donor",
+        "taxa",
+        "biosample_term",
+        "classification"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "accession",
+        "alternate_accessions",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "sample.json#/properties"
+        },
+        {
+            "$ref": "biosample.json#/properties"
+        },
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        },
+        {
+            "$ref": "mixins.json#/product_info"
+        },
+        {
+            "$ref": "mixins.json#/url"
+        },
+        {
+            "$ref": "mixins.json#/taxa"
+        }
+    ],
+    "dependentSchemas": {
+        "$merge": [
+            "biosample.json#/dependentSchemas"
+        ],
+        "passage_number": {
+            "comment": "Only in vitro system with classification as a cell line are allowed to have passage_number property.",
+            "properties": {
+                "classification": {
+                    "enum": [
+                        "cell line"
+                    ]
+                }
+            }
+        }
+    },
+    "dependentRequired": {
+        "time_post_factors_introduction": [
+            "time_post_factors_introduction_units"
+        ],
+        "time_post_factors_introduction_units": [
+            "time_post_factors_introduction"
+        ]
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "classification": {
+            "title": "Classification",
+            "description": "The general category of this type of in vitro system.",
+            "comment": "Required property. The specified category should match the ontology used to describe this type of in vitro system.",
```

Comment:
"should match the ontology type used"

Reply:
I think providing an example in the comment would be helpful as well. For example "brain organoid" using UBERON ontology.

Reply:
cell line using CL ontology

Reply:
done